third_party/amd/include/Dialect/TritonAMDGPU/IR/TritonAMDGPUOps.td

```diff
@@ -152,6 +152,22 @@ def InstructionSchedHint : TT_AMDGPU_Op<"instruction_sched_hint", []> {
   let assemblyFormat = [{ attr-dict }];
 }
 
+def condBarrierOp :
+  TT_AMDGPU_Op<"cond_barrier">,
+  Arguments<(ins I1:$pred)> {
+  let summary = "set barrier conditionally";
```

Comment:
Nit: "Conditionally set barriers to synchronize partial threads in a block"

Reply:
Will address it in the new PR